provider/marathon/marathon.go

```diff
@@ -420,6 +444,193 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type serviceProperties map[string]string
```

Comment:
Since this is a map from property (names) to values, how about calling it `servicePropertyValues`?